user/conditions-v1.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Conditions
+layout: en
+---
+
+> This page documents the behavior of conditions `v1`, which is currently
+> available using `conditions: v1` in your `.travis.yml` file. This new
+> version replaces `v0` which is currently the default version, and in the
+> process of being deprecated in favor of `v1`. The documentation for `v0` can
+> be found [here](/user/conditions-v0).
+
+Conditions can be used to filter out, and reject builds, stages, and jobs by
+specifying conditions in your build configuration (your `.travis.yml` file).
+See [Conditional Builds, Stages, and Jobs](/user/conditional-builds-stages-jobs)
+for details.
+
+## Examples
+
+This condition requires the branch name to be master (note for PRs this is the
+base branch name):
+
+```
+branch = master
+```
+
+This requires the tag name to match a regular expression (enclose in slashes
+for more complicated expressions):
+
+```
+tag =~ ^v1
+tag =~ /^(v1|v2)/
+```
+
+This requires the event type to be either `push` or `pull_request`:
```

Comment:
By event type we really mean build type, right?

Reply:
jup. we talk about event types internally, but i've changed it to build type now.